source/developers/file-transfer-protocol.rst

```diff
@@ -0,0 +1,226 @@
+
+Prerequisites
+-------------
+
+- MinIO RELEASE.2023-04-20T17-56-55Z or later.
+- Enable an FTP or SFTP port for the server.
+- A port to use for the FTP commands and a range of ports to allow the FTP server to request to use for the data transfer.
+
+Procedure
+---------
+
+1. Start MinIO with an FTP and/or SFTP port enabled.
+
+   .. code-block:: shell
+      :class: copyable
+
+      minio server http://server{1...4}/disk{1...4}        \
+      --ftp="address=:8021"                                \
+      --ftp="passive-port-range=30000-40000"               \
+      --sftp="address=:8022"                               \
+      --sftp="ssh-private-key=/home/miniouser/.ssh/id_rsa" \
+      ...
+    
+   See the :mc-cmd:`minio server --ftp` and :mc-cmd:`minio server --sftp` for details on using these flags to start the MinIO service.
+   To connect to the an ftp port with TLS (``FTPS``), pass the ``tls-private-key`` and ``tls-public-cert`` keys and values, as well, unless using the MinIO default TLS keys.
+
+   The output of the command should returns a response that includes something like the following:
+
+   .. code-block:: shell
+
+      MinIO FTP Server listening on :8021
+      MinIO SFTP Server listening on :8022
+
+2. Use your preferred ftp client to connect to the MinIO deployment with the user whose policies you want to use to access buckets and objects.
+
+   The specifics of connecting to the MinIO deployment depend on your FTP client.
+   Refer to the documentation for your client.
+
+   To connect over TLS or through SSH, you must use a client that supports the desired protocol.
+
+Examples
+--------
+
+The examples here use the ``ftp`` CLI client on a Linux system.
+
+Connect to an FTP Server
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following example connect to a server using ``minio`` credentials to list contents in a bucket named ``runner``
+
+.. code-block:: shell
+
+   > ftp localhost -P 8021
+   Connected to localhost.
+   220 Welcome to MinIO FTP Server
+   Name (localhost:user): minioadmin
+   331 User name ok, password required
+   Password:
+   230 Password ok, continue
+   Remote system type is UNIX.
+   Using binary mode to transfer files.
+   ftp> ls runner/
+   229 Entering Extended Passive Mode (|||39155|)
+   150 Opening ASCII mode data connection for file list
+   drwxrwxrwx 1 nobody nobody            0 Jan  1 00:00 chunkdocs/
+   drwxrwxrwx 1 nobody nobody            0 Jan  1 00:00 testdir/
+   ...
+
+Allow MinIO to Connect to an FTP Server over TLS (``FTPS``)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following example starts MinIO with ``FTPS`` enabled.
+
+.. code-block:: shell
+   :class: copyable
+
+   minio server http://server{1...4}/disk{1...4} \
+   --ftp="address=:8021"                         \
+   --ftp="passive-port-range=30000-40000"        \
+   --ftp="tls-private-key=path/to/private.key"   \
+   --ftp="tls-public-cert=path/to/public.crt"    \
+   ...
```

Comment:
Maybe add here "Omit the ``tls-private-key`` and ``tls-public-cert`` parameters to use the MinIO default TLS keys for FTPS. See <network> for more information on MinIO TLS keys.